com.unity.render-pipelines.core/Runtime/Utilities/Blit.hlsl

```diff
@@ -0,0 +1,209 @@
+#ifndef UNITY_CORE_BLIT_INCLUDED
+#define UNITY_CORE_BLIT_INCLUDED
+
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl"
+#include "Packages/com.unity.render-pipelines.core/ShaderLibrary/UnityInstancing.hlsl"
+
+TEXTURE2D_X(_BlitTexture);
+TEXTURECUBE(_BlitCubeTexture);
+SamplerState sampler_PointClamp;
+SamplerState sampler_LinearClamp;
+SamplerState sampler_PointRepeat;
+SamplerState sampler_LinearRepeat;
+uniform real4 _BlitScaleBias;
+uniform real4 _BlitScaleBiasRt;
+uniform real _BlitMipLevel;
+uniform real2 _BlitTextureSize;
+uniform uint _BlitPaddingSize;
+uniform int _BlitTexArraySlice;
+
+#if SHADER_API_GLES
+struct Attributes
+{
+    real4 positionCS       : POSITION;
```

Comment:
positionOS

Reply:
I suppose both are true, since the object space is clip space in this case. Anyway, i changed it back like it was originally.